.travis.yml

```diff
@@ -67,15 +67,6 @@ addons:
       - zlib1g-dev
       # required for Qt 5.9 from 'beineri' PPA
       - libgl1-mesa-dev
-  homebrew:
-    update: true
```

Comment:
This should have been equivalent to `brew update`. I don't know why it makes it take longer.
This PR looks good.
I wonder why we do `brew update` at all. Maybe just list the packages here and be done with it?

Reply:
>I wonder why we do brew update at all. Maybe just list the packages here and be done with it?

Since our home-brewed libtorrent package is dependent on boost, the brew update will be triggered eventually unless you can maintain a specific version that will follow the Travis CI default installed version.
source: https://github.com/Chocobo1/qBittorrent/commit/a6ce3e4ea4e63702954bec062f727369b63f4834
build: https://travis-ci.org/Chocobo1/qBittorrent/jobs/520605634#L318

Another thing, homebrew recently has updated libtorrent to 1.2 so we need to provide 1.1.x for our TravisCI builds, I'll prefer to merge this PR first and you can make the changes later.
See: https://travis-ci.org/qbittorrent/qBittorrent/jobs/520587016#L990